docs/api/dialog.md

```diff
@@ -175,6 +175,17 @@ it is usually used to report errors in early stage of startup.  If called
 before the app `ready`event on Linux, the message will be emitted to stderr,
 and no GUI dialog will appear.
 
+### `dialog.showCertificateTrustDialog(browserWindow, certificate, message, callback)` _macOS_
```

Comment:
If/when this gets support on other platforms there might be some additional options needed so it might be better to start this off as:

```js
dialog.showCertificateTrustDialog(browserWindow, options, callback)
```

You can leave the C++ signature as-is, and just pluck the options in JS like `showMessageBox` does.